addons/binding/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -0,0 +1,104 @@
+| refresh  | 30      | No       | Time between to API requests in minutes. Minimum 15 minutes. |
+| warningCount | 1   | No       | Number of warnings to provide. For each warning there will multiple channels.| 
+
+The plugin will deliver no warnings if the number of retrieved warnings for one thing is to big. This can only happen if you select to many Cells with the %-Operator. It will happen if the area contains about 300+ warnings.
+
+Example:
+
+```
+dwdunwetter:dwdwarnings:cologne "Warnings Cologne" [ cellId="105315000", refresh=15, warningCount=1 ]
+```
+
+
+## Channels
+
+The are multiple channlöes for every weather warning. The channels are numbered, so channels ending in 1 are for the first warning, ending with 2 are for the second warning and so on.
+The warnings will be sorted by severity and for warnings with same severity by the valid from date. This ensures that in the channels for the first warning will always be the highest severity.
+If the API returns more warnings than configured number in the thing, the warnings with the lowest severity will be dropped.  
+ 
+| Channel | Type | Beschreibung |
+| ------- | ---- | ------------ |
+| warningN | Switch | ON if a warning is present, OFF else |
+| UpdatedN | Trigger Channel | Triggers NEW if a warning is send the first time |
+| severityN | String | Severity of the warning. Possible values are Minor, Moderate, Severe and Extreme |
+| headlineN | String | Headline of the warning like "Amtliche Warnung vor FROST" |
+| descriptionN | String | Textual description of the warning |
+| eventN | String | Type of the warning, e.g. FROST |
+| effectiveN | DateTime | Issued Date and Time |
+| onsetN | DateTime | Start Date and Time for which the warning is valid |
+| expiresN | DateTime | End Date and Time for which the warning is valid |
+| altitudeN | Number:Length | Lower Height above sea level for which the warning is valid |
+| ceilingN | Number:Length | Upper Height above sea level for which the warning is valid |
+| urgencyN | String| Urgency of the warning Possible values are Future and Immediate |
+| instructionN | String | Additional instructions and safety informations |
+
+All channels are readonly!
+
+The main purpose of the channel _warning_ is to be used for controlling visibility in sitemaps. Or the check in rules if there is warning present. He should not be used for rules that need to fire if a new warning shows up. If a warning is replaced by another warning, that channels stays at ON, there will be no state change. For rules the need to fire if a new warning show up there is the trigger channel _updatedN_. That trigger channels fires the event whenever a warning is send the first time to that thing. That channel triggers if a warning is replaced by another.
```

Comment:
Can you start each sentence on a new line in the READMEs? That way it's easier to add review comments.

Reply:
Sure, I fixed some typos and spelling mistakes also